### PR TITLE
Remove openfast@fsi for Sunspot

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

File: configs/base/packages.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ packages:
   tioga:
     version: [develop]
   openfast:
-    version: [develop]
```
Comment (psakievich) [inline configs/base/packages.yaml:21]: Does 3.5.3 have the fsi features?  I was under the impression they wouldn't be coming until 4.0.

Comment (jrood-nrel) [reply configs/base/packages.yaml:21]: oh maybe not

Comment (jrood-nrel) [reply configs/base/packages.yaml:21]: I will just close this. I think you're right.

**Comment by jrood-nrel** [comment]
Actually we can merge this change for Sunspot at least.

**Review by psakievich — APPROVED** [review]

**Comment by psakievich** [comment]
We might be okay to default to Openfast 3.5, just want to make sure we don't require it on systems where we are doing fsi. But I don't think you did a requires anyways. So it was probably fine

**Pull request merged**